Templates/QueueTrigger-Python/function.json

```diff
@@ -7,7 +7,7 @@
       },
       {
         "type": "table",
```

Comment:
All of the json files have an extra level of unecessary indenting. Anyone know why, or whose editor is doing this?